python/example_robot_data/robots_loader.py

```diff
@@ -458,6 +474,52 @@ class PandaLoader(RobotLoader):
     ref_posture = "default"
 
 
+class PandaLoaderCollision(PandaLoader):
+    urdf_filename = "panda_collision.urdf"
+    srdf_filename = "panda.srdf"
+
+    def __init__(self, verbose=False):
+        super().__init__(verbose=verbose)
+
+        cmodel = self.robot.collision_model.copy()
+        list_names_capsules = []
+        # Iterate through geometry objects in the collision model
+        for geom_object in cmodel.geometryObjects:
+            geometry = geom_object.geometry
+            base_name = geom_object.name[
+                :-2
+            ]  # Remove the last characters from the name
```

Comment:
Your goal is not to remove the last characters: you are explaining your solution instead of explaining your problem in that comment.

Your goal is to remove the last underscore and the numbers after that. And if those numbers are >9, your solution won't work. Here is a more explicit version:

```suggestion
            base_name = '_'.join(geom_object.name.split('_')[:-1])
```

A regex would be even better, but I think this version is easier to read.
Your comment should include what you explained to me: your goal is to remove what looks like a suffix with an index of the different bodies attached there.